src/main/java/com/ssoyeong/studyapplication/controller/TestController.java

```diff
@@ -1,19 +1,26 @@
 package com.ssoyeong.studyapplication.controller;
 
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 public class TestController {
+    private static String sayHello = "Hello, %s!";
```

Comment:
```suggestion
    private static final String GREETING_MESSAGE = "Hello, %s!";
```

`static`과 `final` 키워드에 대해서 알아보기.